src/index.js

```diff
@@ -1 +1,41 @@
-console.log('Hello');
\ No newline at end of file
+const fs = require('fs');
+const path = require('path');
+const csv = require('csvtojson');
+
+const main = async () => {
+    const txtDir = path.join(__dirname, '../', 'txt');
+    const txtFilePath = path.join(__dirname, '../', 'txt/txt-result.txt');
```

Comment:
Consider extracting hard-coded strings into separate constants at the top of the file.

Reply:
I've extracted constants into their own file.